### ISSUE TITLE
"Get in touch" link should be redirected to proper url

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-org#3009]
It looks like the link on the [a11y fundamentals page](https://www.patternfly.org/v4/accessibility/accessibility-fundamentals), "get in touch" is directing to an old page. I'm guessing that it should be updated to the community page?
